mlir/lib/Dialect/Vector/IR/VectorOps.cpp

```diff
@@ -3627,27 +3648,21 @@ LogicalResult InsertStridedSliceOp::verify() {
 }
 
 namespace {
-/// Pattern to rewrite an InsertStridedSliceOp(SplatOp(X):src_type,
-/// SplatOp(X):dst_type) to SplatOp(X):dst_type.
+/// Rewrite insert_strided_slice(splat-like(v), splat-like(v)) as v
```

Comment:
```suggestion
/// Rewrite insert_strided_slice(splat-like(v), splat-like(v)) as v.
```
See https://llvm.org/docs/CodingStandards.html#commenting